src/librustc_metadata/encoder.rs

```diff
@@ -490,7 +490,11 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
             } else {
                 None
             },
-
+            proc_macro_stability: if is_proc_macro {
+                tcx.lookup_stability(DefId::local(CRATE_DEF_INDEX)).map(|stab| stab.clone())
```

Comment:
please add a test that shows that `HashStable` is unstable

Reply:
Added a test.